frontend/src/components/errors/assets/NotFoundIllustration.tsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react'
+
+import theme from 'common/theme'
+
+export default function NotFoundIllustratio() {
```

Comment:
There's a typo in the name of the component

```suggestion
export default function NotFoundIllustration() {
```